app/api/v1/endpoints/insurance.py

```
@@ -0,0 +1,65 @@
+from uuid import UUID
+from app.db.session import get_db
+from app.schemas.insurance import InsuranceSchema, InsuranceCreate, InsuranceUpdate
+from app.crud import insurance as insurance_crud
+
+router = APIRouter()
+
+@router.get("/", response_model=List[InsuranceSchema])
+async def read_insurances(
+    skip: int = 0,
+    limit: int = 100,
+    db: AsyncSession = Depends(get_db)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)